static/css/devhub/new-landing/base.less

```diff
@@ -3,8 +3,9 @@
 
 // Grid
 @grid-max: 940px;
-@small: ~"only screen and (max-width: 599px)";
-@medium: ~"only screen and (min-width: 600px)";
+@extra-small: ~"only screen and (max-width: 599px)";
+@small: ~"only screen and (min-width: 599px)";
```

Comment:
This puts small/medium back to what they were when I started but I've added a extra-small for a bit more granularity.